apps/server/src/shared/infra/collaborative-storage/strategy/nextcloud/nextcloud.client.spec.ts

```diff
@@ -260,7 +260,7 @@ describe('NextCloud Adapter Strategy', () => {
 
 			// Act & Assert
 			await expect(
-				client.setGroupPermissions(testGroupId, testFolderId, [true, true, true, true, true])
+				client.setGroupPermissions(testGroupId, testFolderId, [true, false, true, true, false])
```

Comment:
why?

Reply:
that way we could test a non tested branch in the coverage